components/raftstore/src/store/config.rs

```diff
@@ -137,6 +137,17 @@ pub struct Config {
     #[online_config(skip)]
     pub snap_apply_batch_size: ReadableSize,
 
+    // used to periodically check whether schedule pending applies in region runner
+    #[doc(hidden)]
+    #[online_config(skip)]
+    pub region_worker_tick_interval: ReadableDuration,
```

Comment:
Seems that the name is changed from "pending_apply_check_xxx". The previous name is more obervious to me IMO, could you explain your idea behind this name change?

Reply:
We already have this configuration in proxy long ago, so this is more like a backport to TiKV

Reply:
@busyjay seems that this name has already been used in ser/deser config items. Do you have some suggestions?